hummingbot/connector/exchange/bittrex/bittrex_exchange.pyx

```diff
@@ -508,7 +526,7 @@ cdef class BittrexExchange(ExchangeBase):
 
                     if executed_amount_diff > s_decimal_0:
                         self.logger().info(f"Filled {executed_amount_diff} out of {tracked_order.amount} of the "
-                                           f"{order_type_description} order {tracked_order.client_order_id}.")
+                                           f"{order_type_description} order {tracked_order.client_order_id}. - ws")
```

Comment:
What is the `- ws` for?

Reply:
To differentiate from the same message posted in the REST API verification loop